### PR TITLE
explain D on first page

### PR DESCRIPTION
**Pull request by wilzbach**
I think we should explain D to the reader - text is taken from 
http://dlang.org/overview.html

### CODE REVIEW

File: public/content/en/welcome.md [hunk 1]
```diff
@@ -7,6 +7,16 @@ This tour will give you an overview of this powerful and expressive systems
 programming language which compiles directly to efficient, *native*
 machine code.
 
+## What is D
```
Comment (stonemaster) [inline public/content/en/welcome.md]: 3rd level section needed. Build failed because of it.

File: public/content/en/welcome.md [hunk 2]
```diff
@@ -7,6 +7,31 @@ This tour will give you an overview of this powerful and expressive systems
...
 
+### What is D
+
+D has many [unique features](http://dlang.org/overview.html) and is a
+general purpose systems and applications programming language.
+It is a _high level language_, but retains the ability to write _high
+performance_ code and interface directly with the _operating system API's_
+and with hardware.
+D is well suited to writing medium to large-scale programs with many million
+lines of code.
```
Comment (wilzbach) [inline public/content/en/welcome.md]: I am not sure how well "million lines of code" sounds, do you have a better idea?

**Comment by wilzbach** [comment]
rebased, reformatted (<80 char per lines) and added highlighting and reworked a bit ;-)

File: public/content/en/welcome.md (hunk 2 above)
Comment (stonemaster) [inline public/content/en/welcome.md]: I would just drop the "with .." sentence and replace it with "D is well suited to writing medium to large-scale programs". I think the reader will know what we mean by that.

**Comment by stonemaster** [comment]
@wilzbach ping

**Comment by wilzbach** [comment]
> @wilzbach ping

sorry moved from one country to the other - thanks for the ping :)

File: public/content/en/welcome.md (hunk 2 above)
Comment (wilzbach) [inline public/content/en/welcome.md]: removed ;-)

**Pull request merged**